addons/mail/tests/test_mail_render.py

```diff
@@ -330,62 +304,35 @@ def test_template_rendering_various(self):
         # static xml
         src = '<p class="text-muted"><span>This is a string</span></p>'
         expected = '<p class="text-muted"><span>This is a string</span></p>'
-        for engine in ['jinja', 'qweb']:
+        for engine in ['inline_template', 'qweb']:
             result = MailRenderMixin._render_template(
                 src, partner._name, partner.ids, engine=engine,
             )[partner.id]
-            self.assertEqual(expected, result if engine == 'jinja' else result)  # tde: checkme
+            self.assertEqual(expected, result)  # tde: checkme
 
         # code xml
         srces = [
-            '<p class="text-muted"><span>This is a string with a number ${13+13}</span></p>',
+            '<p class="text-muted"><span>This is a string with a number {{ 13+13 }}</span></p>',
             '<p class="text-muted"><span>This is a string with a number <t t-out="13+13"/></span></p>',
         ]
         expected = '<p class="text-muted"><span>This is a string with a number 26</span></p>'
-        for engine, src in zip(['jinja', 'qweb'], srces):
+        for engine, src in zip(['inline_template', 'qweb'], srces):
             result = MailRenderMixin._render_template(
                 src, partner._name, partner.ids, engine=engine,
             )[partner.id]
-            self.assertEqual(expected, result if engine == 'jinja' else result)  # tde: checkme
-
-        # condition block xml
```

Comment:
All that stuff should probably be coming back, somehow. Also to check with @std-odoo

Reply:
As we do not have code blocs in inline_template, I don't see why we should have those tests.

Reply:
Could be kept for qweb at least, even if a bit redundant, this PR should keep tests as untouched as possible (will clean post merge if necessary)

Reply:
The above code marked as removed in this diff is actually still there, the diff is missleading.
I just changed `self.assertEqual(expected, result if engine == 'jinja' else result)` to `self.assertEqual(expected, str(result))`.

The below code is still irrelevant as we do not have code blocks in inline template. What do you want me to test?